resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -357,6 +387,36 @@ private[spark] class ExecutorPodsAllocator(
     }
   }
 
+  private def replacePVCsIfNeeded(
+      pod: Pod,
+      resources: Seq[HasMetadata],
+      reusablePVCs: mutable.Buffer[PersistentVolumeClaim]) = {
+    val replacedResources = mutable.ArrayBuffer[HasMetadata]()
```

Comment:
Set instead ?

Reply:
Ya, maybe. I didn't try to add another semantic like adding uniqueness here.

Reply:
The only use of `replacedResources` is to do a `filterNot` on `resources` in this method - right ?
So whether unique or not does not matter ?

Reply:
Yes, that's the point. It doesn't matter. So, I used `ArrayBuffer` simply to keep the input order.

Reply:
You are right~